src/UI/class-admin-warning.php

```diff
@@ -73,6 +73,10 @@ private function should_display_admin_warning(): bool {
 			return false;
 		}
 
+		if ( defined( 'is_managed' ) && true === constant( 'is_managed' ) ) {
```

Comment:
I thought we were going to return `is_managed` value in `wp_parsely_credentials` filter as an array key/value pair .... or is it not the case?

Reply:
If possible kindly add tests for this also.

Reply:
About the test, please see the "How has this been tested?" section. Noting this in #1161 though, so we can consider doing this in the future.

About the Parse.ly `wp_parsely_credentials` filter returning an is_managed value, there were no such plans or discussions to my knowledge.

Isn't `is_managed` a PHP constant that gets set in the platform side? If not, we'll need to revise our approach.

Reply:
As far as I know `is_managed => true` will be return via `wp_parsely_credentials` filter if the plugin is managed by Platform. cc: @mjangda for double check if my understanding is wrong.

Reply:
Noting that there are 2 "managed" aspects in the plugin: `credentials` and `options` which are governed by their respective filters. Credentials and filters behave differently, and thus we've got 2 different filters to manage them at the moment (they could possibly be managed by a single filter, but this would be a refactoring).

If we use the `wp_parsely_credentials` filter, the passed credentials will be in managed mode (read-only or hidden). Same for options with `wp_parsely_managed_options`. This allows any combination of settings, and allows different behaviors if the platform sets different filters for different user types or situations.

Reply:
`is_managed => true` will only be return by `wp_parsely_credentials` and plugin should have been able to use this for any purpose.

Reply:
This should the address the issue in this PR:

```suggestion
		if ( $this->parsely->are_credentials_managed ) {
```

To Mehmood's point though, we have a separate issue described here: https://github.com/Automattic/vip-parsely/issues/53 (which can be fixed in a separate PR).

Basically, there will be cases where we have a site that we expect to have managed credentials but the credentials will not be generated yet. And we want to restrict that site's ability to manage the credentials until they are generated. Currently, `are_credentials_managed` only works if we pass in valid values. So, the idea is to be able to set another flag that tells the plugin that this site has its credentials managed, so don't expose things related to that (form controls, warnings, etc.) that customers can't action.

Reply:
OK, this makes sense. I'd examine this flag as above in the function that sets the `are_credentials_managed` value, so we have a single check for everything. Would this work for everybody?

Reply:
Works for me (but can sure after viewing the changes)

Reply:
Available in https://github.com/Parsely/wp-parsely/pull/1790.